Ginger/Ginger/UserControlsLib/ucGridView/ucGrid.xaml.cs

```diff
@@ -1490,6 +1490,9 @@ public static DataTemplate GetDataColGridTemplate(string Path)
                 combo.SetValue(ComboBox.SelectedIndexProperty, 0);
             }
 
+            if (comboSelectionChangedHandler != null)
```

Comment:
for what this is needed?

Reply:
for the ComboBox style you asked to use over Wizard compare part. to bind the selection changed event. That's available only on this functions overloaded one.